
# Se supone que el script da como resultado el conseno de angulo 1, 
# pero en lugar de eso, lanza una error. Rapara el error.

'''
import math

print(math.cosine(1))
'''

from math import cos

print(cos(1))